### ISSUE TITLE
Move to an organization account?

### ISSUE THREAD

**Issue by cxw42** [rain-1/linenoise#31]
This is a fantastic library, and I appreciate your work merging fixes and updating the codebase.

I asked @antirez in antirez/linenoise#155 if he would be willing to consider moving this to an organization account.  If so, would you be amenable to pushing your fixes upstream?  I would be delighted to host linenoise in <https://github.com/interpreters> if you or @antirez didn't want to create your own organization.  Access rights and everything stay the same, plus other people can be granted write access.

Thanks for considering this request!

**Comment by rain-1** [comment]
Yes, would be very happy to merge upstream. I think antirez is busy with other stuff at the moment but I aimed to keep everything (except the UTF8 code) in the original spirit of linenoise hopefully it can be upstreamed.

Appreciate your interest.
